src/table.rs

```diff
@@ -8,8 +28,17 @@ pub struct Parameters {
 impl_drop!(Parameters, osrmc_sys::osrmc_table_params_destruct);
 
 impl Parameters {
-    pub fn new() -> Result<Parameters> {
+    pub fn new(annotations: Annotations) -> Result<Parameters> {
         let handle = call_with_error!(osrmc_table_params_construct())?;
+
+        unsafe {
+            osrmc_sys::osrmc_table_params_set_annotations(
+                handle,
+                annotations.has_distance(),
```

Comment:
We define `Annotations` to be bitflags, but then pass them as bools. Why not just pass bitflags?

Reply:
There's no typesafe way to do that through C, so it's better to pass them as bools there.

Reply:
Although they actually are the wrong way around here. 🙈 

I'm changing the API for `libosrmc` based on some feedback and will update this.

Reply:
I've removed this from the API of `osrm-rs`, as part of reworking `libosrmc`. I think we can figure out the API for this if we ever need to _disable_ distances in the returned Table responses.